src/go/rpk/pkg/cli/cmd/debug/bundle_linux.go

```diff
@@ -37,15 +37,62 @@ import (
 	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
 	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
 	osutil "github.com/redpanda-data/redpanda/src/go/rpk/pkg/os"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
 	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/system"
 	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/system/syslog"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/afero"
 	"github.com/twmb/franz-go/pkg/kadm"
 	"github.com/twmb/franz-go/pkg/kgo"
+	"golang.org/x/sys/unix"
 	"gopkg.in/yaml.v3"
 )
 
+// processFilepath will process the given path and sets:
```

Comment:
What do you think about `determineFilepath`? `process` makes me think this is doing something to an existing filepath.